docs/general/adapters.md

```diff
@@ -194,6 +194,43 @@ The user could pass in `include=**`.
 
 We recommend filtering any user-supplied includes appropriately.
 
+#### Include Parameters
+
+You may wish to include related resources based on the `include` request
+parameter (See [fetching_includes](http://jsonapi.org/format/#fetching-includes). You can opt-in to this pattern:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  associations_via_include_param(true)
```

Comment:
Re: that PR/issue, I think we need to discuss what to do about it, or (if everyone's OK with it, one of us could just fix 'the problem') , but I know there was a desire to re-architect relationship building. And I think there may have been some blocker to that work? I don't remember